Issue with JSON parsing

Hi! This script is a great start and I hope you continue to develop it.

When I ran the script, I got the following:
```bash
> make                                                                                                
osascript notes.applescript > notes.txt
rm -rf dist
mkdir dist
python3 notes.py notes.txt
Traceback (most recent call last):
  File "/.../apple-notes-exporter/notes.py", line 56, in <module>
    "creation_date": get_datetime_string(note_creation_date),
  File "/.../apple-notes-exporter/notes.py", line 20, in get_datetime_string
    dt_obj = datetime.strptime(string, DATETIME_IMPORT_MASK)
  File "/usr/local/Cellar/python@3.9/3.9.13_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/Cellar/python@3.9/3.9.13_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data 'Tuesday, June 14, 2016 at 21:55:08' does not match format '%A, %B %d, %Y at %I:%M:%S %p'
make: *** [json] Error 1
```

It looks like the failing note is using 24h time so it is missing the `%p` format from `DATETIME_IMPORT_MASK`

--- a/notes.py
+++ b/notes.py
@@ -13,11 +13,17 @@ OUTPUT=os.path.join(HERE, "dist")
 # Mask to parse:
 # Friday, September 22, 2017 at 1:00:52 AM
 DATETIME_IMPORT_MASK='%A, %B %d, %Y at %I:%M:%S %p'
+DATETIME_IMPORT_MASK_24H='%A, %B %d, %Y at %H:%M:%S'
 DATETIME_SAVE_MASK='%Y-%m-%dT%H:%M:%SZ'
 
 
 def get_datetime_string(string):
-    dt_obj = datetime.strptime(string, DATETIME_IMPORT_MASK)
+    try:
+        dt_obj = datetime.strptime(string, DATETIME_IMPORT_MASK)
+    except ValueError:
+        # 24h time
+        dt_obj = datetime.strptime(string, DATETIME_IMPORT_MASK_24H)
+
     return dt_obj.strftime(DATETIME_SAVE_MASK)
 
 

--- a/notes.py
+++ b/notes.py
@@ -10,16 +10,17 @@ SEPARATOR='###|'
 HERE=os.path.dirname(os.path.realpath(__file__))
 OUTPUT=os.path.join(HERE, "dist")
 
-# Mask to parse:
-# Friday, September 22, 2017 at 1:00:52 AM
-DATETIME_IMPORT_MASK='%A, %B %d, %Y at %I:%M:%S %p'
+# Example mask to parse:
+# 12H time: `Friday, September 22, 2017 at 1:00:52 PM`
+# 24H time: `Friday, September 22, 2017 at 13:00:52`
+DATETIME_IMPORT_MASK_12H='%A, %B %d, %Y at %I:%M:%S %p'
 DATETIME_IMPORT_MASK_24H='%A, %B %d, %Y at %H:%M:%S'
 DATETIME_SAVE_MASK='%Y-%m-%dT%H:%M:%SZ'
 
 
 def get_datetime_string(string):
     try:
-        dt_obj = datetime.strptime(string, DATETIME_IMPORT_MASK)
+        dt_obj = datetime.strptime(string, DATETIME_IMPORT_MASK_12H)
     except ValueError:
         # 24h time
         dt_obj = datetime.strptime(string, DATETIME_IMPORT_MASK_24H)

